operator/README.md

```diff
@@ -0,0 +1,201 @@
+        scanPoliciesDir: '/zap/policies'
+        scanPolicyName: 'API-minimal-example'
+  
+        # passive scan
+        disabledPassiveScan: "2,10015,10027,10096,10024"
+        # https://www.zaproxy.org/docs/alerts/
+        # 2: Private IP Disclosure
+        # 10015: Incomplete or No Cache-control Header Set
+        # 10027: Information Disclosure - Suspicious Comments
+        # 10096: Timestamp Disclosure
+        # 10024: sensitive info in URL
+  
+      #### AUTHENTICATION ####
+      # Define authentication method for the context. Possible values are:
+      # "null", "manualAuthentication"; "scriptBasedAuthentication"; "httpAuthentication";
+      # "formBasedAuthentication"
+      authMethod: null
+  
+      #### UPSTREAM PROXY ####
+      proxy:
+        useProxyChain: False
+        proxyAddress: ''
+        proxyPort: ''
+        skipProxyAddresses: ('127.0.0.1;', 'localhost')
+  image:
+    pullPolicy: Always
+    repository: quay.io/redhatproductsecurity/rapidast
+    tag: latest
+  job:
+    cron: false
+    schedule: '* * * * *'
+  pvc: rapidast-pvc
+```
+
+### RapiDAST Config
+
+Most of the application specific scan configuration exists under `spec.config`. This is itself YAML syntax included here as a multiline string. This is the same configuration used in the core RapiDAST project [here](https://github.com/RedHatProductSecurity/rapidast/blob/development/config/config-template-local.yaml)
+
+#### Operator Config Options
+There are additional config options specific to the operator.
+
+##### image
+Under image, you can change the pull policy, repository, and tag for the rapidast image. Unless testing changes, these are best left to their default values.
+
+##### job
+Under job, there are two options. 
+- cron: This is a boolean value. If true, will run RapiDAST scans on the specified schedule
+- schedule: Defines the schedule used if cron is set to true. This uses 
+
+##### pvc
+The `pvc` specifies the persistent volume claim to use. This is used by the operator to store results. If the specified PVC does not exist, it will be created. Note that this will not be uninstalled should you delete the RapiDAST resource, and will need to be explicitly deleted by itself when you are sure you don't need any of the data contained on the bound volume.
+
+### Apply RapiDAST Resource
+
+To apply the RapiDAST resource to the cluster you are already logged into, run
+
+`oc apply -f rapidast.yaml`
+
+where `rapidast.yaml` is modified to the filename used for your RapiDAST CR.
+
+#### Demo application
+An application for demonstration purposes is available [here](https://github.com/jpweiser/rapitester). You can use this application, and the provided configuration to see the RapiDAST operator at work.
+
+## Getting Results
+
+The easiest way to get results is to use a pod that mounts the same PVC used to store the results, and use `oc rsync POD:/results_dir local_dir` to copy an entire directory, or `oc cp POD:/path/to/file /local/path`
```

Comment:
as discussed, use kubectl instead of oc

Reply:
Might need to look at this one a bit more. I don't think the rsync option exists in kubectl, thought cp should. This will need to be tested.